tests/src/kakarot/instructions/test_environmental_information.py

```diff
@@ -113,6 +116,25 @@ def test_gasprice(self, cairo_run):
             cairo_run("test__exec_gasprice")
 
     class TestExtCodeHash:
+        def _pack_into_u64_words(self, bytecode):
+            bytes8_little_endian = [
+                int.from_bytes(bytes(bytecode[i : i + 8]), byteorder="little")
+                for i in range(0, len(bytecode), 8)
+            ]
+            last_word_bytes_used = len(bytecode) % 8
+            last_word = (
+                bytes8_little_endian[-1]
+                if (bytes8_little_endian and last_word_bytes_used != 0)
+                else 0
+            )
+
+            full_words = []
+            if len(bytecode) % 8 == 0:
+                full_words = bytes8_little_endian
+            else:
+                full_words = bytes8_little_endian[:-1]
+            return len(full_words), full_words, last_word, last_word_bytes_used
```

Comment:
is this chat gpt generated? ;) could be simplified

```
if len(bytecode) % 8:
    last_word = bytes8_little_endian[-1]
    full_words = bytes8_little_endian[:-1]
else:
    last_word = 0
    full_words = bytes8_little_endian
```

Reply:
I wrote this 😢

Reply:
it's just that you don't make it visible that there is here one single condition based on the len of the bytecode, leading to either full words or remainders

Reply:
```py
            last_word_bytes_used = len(bytecode) % 8
            last_word = (
                bytes8_little_endian[-1]
                if bytes8_little_endian and last_word_bytes_used
                else 0
            )
            full_words = (
                bytes8_little_endian[:-1]
                if last_word_bytes_used
                else bytes8_little_endian
            )
 ```
 
 How about this one ?